src/ai_hawk/llm/llm_manager.py

```diff
@@ -232,6 +250,10 @@ def _create_model(self, config: dict, api_key: str) -> AIModel:
             return OllamaModel(llm_model, llm_api_url)
         elif llm_model_type == GEMINI:
             return GeminiModel(api_key, llm_model)
+        elif llm_model_type == "huggingface":
+            return HuggingFaceModel(api_key, llm_model)
+        elif llm_model_type == "azure":
```

Comment:
Can you move the strings "azure" and "huggingface" to the constants.py file? Make it consistent with the others.

Reply:
Done